charts/opentelemetry-ebpf/values.yaml

```diff
@@ -0,0 +1,275 @@
+# Default values for opentelemetry-ebpf.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+nameOverride: ""
+fullnameOverride: ""
+
+clusterName: ""
+
+images:
+  tag: "v0.10.0"
+  repository: otel
```

Comment:
I think the `reposotory` here should be called `registry`, and the common practice here is to use `registry` as a global variable.